test/test_mpi_tree.jl

```diff
@@ -70,6 +70,15 @@ CI_ON_WINDOWS = (get(ENV, "GITHUB_ACTIONS", false) == "true") && Sys.iswindows()
                             coverage_override=(maxiters = 6,))
     end
 
+    @trixi_testset "elixir_advection_restart_amr.jl" begin
+        @test_trixi_include(joinpath(EXAMPLES_DIR,
+                                     "elixir_advection_restart_amr.jl"),
+                            # Expected errors are exactly the same as in the serial test!
```

Comment:
Do we have the same test in serial?

Reply:
No, but I could easily add it there as well.

Reply:
We don't need it - but we must remove the comment if it's not true to avoid confusion

```suggestion
```